src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -807,6 +809,19 @@ private void handleOptionAnnotation(final Field field, final int fieldPosition)
                 printOrder = fieldPosition * 1000;
             }
 
+            // log a warning if boundaries are set
+            if (optionAnnotation.maxValue() != Double.POSITIVE_INFINITY) {
+                logger.warn("UNSUPPORTED: maximum allowed value for argument --{} is not forced", field.getName());
```

Comment:
I moved this warnings down and use `OptionDefinition.name` to be clearer here. I guess that's what you mean.